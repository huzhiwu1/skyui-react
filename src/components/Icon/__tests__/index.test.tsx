import React from 'react';
import TestRenderer from 'react-test-renderer';
import { Icon } from '../index';

describe('测试Icon', () => {
  it('测试icon能成功渲染', () => {
    const json = TestRenderer.create(<Icon name="sync" />).toJSON();
    expect(json).toMatchSnapshot();
  });
});
