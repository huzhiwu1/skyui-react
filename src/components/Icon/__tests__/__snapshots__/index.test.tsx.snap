// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试Icon 测试icon能成功渲染 1`] = `
<mock-files
  className="sky-icon sky-icon--normal"
/>
`;
